### DISCUSSION TITLE
something similar to foot-server and footclient , is there something like that?

### DISCUSSION THREAD

**Discussion by alfredjophy** [kovidgoyal/kitty#3671] [Q&A]
sometimes the startup is slow for kitty and the ram usage abt 100MB for every instance.
would be easy if there is a kitty-server of sort that can executed at startup and all other be instances of that.
Is there a way to do that?

**Comment by kovidgoyal** [comment]
--single-instance

**Reply by alfredjophy** [reply]
how do i run the single instance in the background at startup?

**Comment by kovidgoyal** [comment]
On Sat, May 29, 2021 at 10:47:54PM -0700, Alfred Jophy wrote:
> how do i run the single instance in the background at startup?

Not sure what you are asking. You run it like you run any other program.

Though you dont actually need to run it startup, that will only speed up
the very first launch of the terminal. Instead just alias kitty to kitty
--single-instance (or edit the desktop file if you use GUI launchers)
